### PR TITLE
Removes unnecessary function and avoids unnecessary calls to updateStats when changing the range on the histogram

### PR DESCRIPTION
**Pull request by saleiva**
CR @javierarce @xavijam 

This was motivated by some debugging I did, where I noticed several repeated calls

### CODE REVIEW

File: src/widgets/histogram/content-view.js [hunk 1]
```diff
@@ -255,8 +254,7 @@ module.exports = cdb.core.View.extend({
 
   _onBrushEnd: function (loBarIndex, hiBarIndex) {
     var data = this._dataviewModel.getData();
-
-    if (!data || !data.length) {
+    if ((!data || !data.length) || (this.model.get('lo_index') == loBarIndex && this.model.get('hi_index') == hiBarIndex)) {
```
Comment (saleiva) [inline src/widgets/histogram/content-view.js]: Avoid executing the function if the new range is the same as the one set previously

**Comment by xavijam** [comment]
Tests are not passing, take a look please. Also, I don't know how exactly histogram with brush works but I have a question. Have you tried this with an example using the state of the url?

**Comment by saleiva** [comment]
Yes yes, reviewing the tests

**Comment by saleiva** [comment]
In local they pass

**Comment by xavijam** [comment]
it is a lint error, please check travis details.

**Comment by xavijam** [comment]
:es:

**Comment by saleiva** [comment]
@javierarce @xavijam doesn't trust me until you review this

**Comment by javierarce** [comment]
Haha, sorry, code looks good to me 👍

**Pull request merged**